scanners/generic/tools/oobtkube.py

```diff
@@ -105,13 +110,35 @@ def get_sarif_output(shared_queue):
     return sarif_conv.convert_to_sarif_json(result_message, artifact_url, snippet)
 
 
-def find_leaf_keys_and_test(data, original_file, ipaddr, port):
+def count_total_leaf_keys(data):
+    count = 0
+    key_list = []
+    for key, value in data.items():
+        if isinstance(value, dict):
+            count += count_total_leaf_keys(value)
+        else:
+            count += 1
+            key_list.append(key)
+
+    logging.info(f"Parameters to be tested: {key_list} (total: {count})")
+    return count
+
+
+# pylint: disable=R0913
+def find_leaf_keys_and_test(
```

Comment:
A comment explaining the purpose of this function would be really helpful

Reply:
Thanks. Added a brief comment.